lib/convict.js

```diff
@@ -349,7 +352,7 @@ var convict = function convict(def) {
      * Exports all the properties (that is the keys and their current values) as JSON
      */
     getProperties: function() {
-      return JSON.parse(JSON.stringify(this._instance));
+      return this._instance;
```

Comment:
What was the reason behind this change?  Seems a little dangerous at first glance, replacing a clone with a direct object reference.